### ISSUE TITLE
Systemd restart and cobbler import

### ISSUE THREAD

**Issue by hyzth** [ContainerSolutions/docker-cobbler#3]
Every time I do _cobbler sync_, it always hangs. Tracing the issue, I found out that during _cobbler sync_, cobbler will try to restart dhcpd. However, no systemd services can be restarted. When I add this line (taken from dockerhub), it works:

> ENV container docker

Do the current code work for you when you do _cobbler sync_?
And another issue, when I do _cobbler import_, it always shows error about kickstart file not found. How do you make cobbler import the distro?

**Comment by thijsschnitger** [comment]
Yes the restart is broken currently, you need to restart dhcpd manually when you want to sync.
I don't recall getting errors while using import. The blog post (see link in the README) explains a bit about what I did. What error do you get? And what steps did you take?

**Comment by hyzth** [comment]
No, I can't even restart dhcpd manually. I got that _cobbler sync_ work by adding the mentioned line. The error is: kickstart file not found. At first I built the container and got into it, then _cobbler check_, restart cobblerd, _cobbler sync_, all worked fine until _cobbler import_.
And I forgot to mention that even with _make import_, I still faced error.

**Comment by thijsschnitger** [comment]
If dhcpd doesn't start then there's something wrong in the config. It's a bit of a PITA to debug, but journalctl -xe should be helpful. For me the issue was using an ip range not served by the network interface I specified, but you might have a different problem.
There should be a cobbler.log somewhere as well, maybe that gives a hint about the kickstart file?
